inbm-lib/inbm_common_lib/utility.py

```diff
@@ -205,7 +205,9 @@ def is_within_directory(directory: str, target: str) -> bool:
     return prefix == abs_directory
 
 
-def safe_extract(tarball: tarfile.TarFile, path: str = ".", members: Optional[Iterable[tarfile.TarInfo]] = None, *, numeric_owner: bool = False) -> None:
+def safe_extract(tarball: tarfile.TarFile,
```

Comment:
Refactor `safe_extract` to use keyword arguments for clarity when calling with default values for `path` and `members`. [medium]